pkg/config/piped.go

```diff
@@ -100,6 +104,11 @@ func (s *PipedSpec) Validate() error {
 			return err
 		}
 	}
+	for _, r := range s.ChartRegistries {
+		if err := r.Validate(); err != nil {
+			return err
+		}
+	}
```

Comment:
this changes not really related to this PR

Reply:
I'll move this change to another PR.